create-repo.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import re
+import subprocess
+import sys
+import textwrap
+
+import requests
+
+sparql_endpoint = 'https://query.wikidata.org/sparql'
+iso_3166_1_code = sys.argv[1].lower()
+
+if os.path.isdir(iso_3166_1_code):
+    raise AssertionError(f'Country repository already exists: {iso_3166_1_code}')
+
+query = f'''\
+SELECT * WHERE {{
+  ?country wdt:P297 '{iso_3166_1_code.upper()}' ;
+           wdt:P37/wdt:P424 ?language ;
+           rdfs:label ?label .
+  FILTER(LANG(?label) = 'en')
+}}
+'''
+
+print(query)
+
+response = requests .post(sparql_endpoint, query,
```

Comment:
Tiny thing, obviously but there's an extraneous space here (I'd have hoped this kind of thing would be picked up by live linting in your editor?)

Reply:
Was using vanilla Atom at the time, which in retrospect wasn't too helpful for Python. Fixed this and other style issues with the PEP8 linting in PyCharm :-)